lib/widget/lists/ventilation_guide_card.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/widgets.dart';
+
+import '../../models/ventilation_guide.dart';
+import '../../style.dart';
+import '../../widget/cards/reusable_card_base.dart';
+import '../../widget/lists/string_list.dart';
+
+class VentilationGuideCard extends StatelessWidget {
+  final VentiliationGuideSection sectionItem;
+  final Color backgroundColor;
+
+  Widget getList(List<String> notes) {
+    if (notes != null && notes.isNotEmpty) {
+      return StringList(
+          items: notes, padding: const EdgeInsets.fromLTRB(8, 0, 0, 0));
+    }
+    return Container();
+  }
+
+  Widget getHeading() {
+    if (sectionItem.heading != null) {
+      return getPaddedItem(
+          Text(
+            sectionItem.heading,
+            style: Styles.textH4,
+          ),
+          const EdgeInsets.fromLTRB(16, 24, 16, 16));
+    }
+    return Container();
+  }
+
+  Widget getPaddedItem(Widget child, EdgeInsets padding) {
+    return Padding(
+        padding: padding ?? const EdgeInsets.fromLTRB(8, 2, 4, 2),
+        child: child);
+  }
+
+  const VentilationGuideCard({this.sectionItem, this.backgroundColor});
+
+  @override
+  Widget build(BuildContext context) {
+    return ReusableCardBase(
+        elevation: 0,
+        color: backgroundColor,
+        child: <Widget>[
+          getHeading(),
+          ...sectionItem.list
+              .map((item) => Padding(
```

Comment:
would be nice to break this (from Padding on down) out into a separate widget or at least a private "builder" function that takes an `item`